client/core/notification.go

```diff
@@ -457,7 +457,7 @@ const TopicBalanceUpdated Topic = "BalanceUpdated"
 
 func newBalanceNote(assetID uint32, bal *WalletBalance) *BalanceNote {
 	return &BalanceNote{
-		Notification: db.NewNotification(NoteTypeBalance, TopicBalanceUpdated, "balance updated", "", db.Data),
```

Comment:
This constructor was an outlier - other `db.Data` notes use empty subject and details, which is the signal for `logNotify` method to skip logging the not before relaying it to consumers. https://github.com/decred/dcrdex/blob/f02dc3d5d2edc1c8437b1b854aaa11051a79d1c0/client/core/notification.go#L48-L49